test/integration/client/client_test.go

```diff
@@ -53,6 +54,10 @@ import (
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/test/integration/framework"
 	imageutils "k8s.io/kubernetes/test/utils/image"
+
+	// the metrics are loaded on cmd/kube-apiserver/apiserver.go
+	// so we need to load them here to be available for the test
+	_ "k8s.io/component-base/metrics/prometheus/restclient"
```

Comment:
this is the problem with metrics, that the are global, so all the tests that run in the same test binary access the same metrics registry.
Since there are other test with metrics I will enable only them here by now, I do not want to do this global import because of the risk of impacting other tests , and then not catch it on presubmit, flakes pointing to this pr, revert, ...